support/elasticsearch2_mapping.erl

```diff
@@ -142,10 +142,18 @@ map_rsc(Id, Context) ->
         es_type => <<"resource">>,
         category => m_rsc:is_a(Id, Context),
         pivot_title => default_translation(proplists:get_value(title, RscProps), Context),
+        pivot_rtsv => map_rtsv(m_edge:objects(Id, Context)),
         incoming_edges => incoming_edges(Id, Context),
         outgoing_edges => outgoing_edges(Id, Context)
     }.
 
+map_rtsv(ObjectIds) ->
+    lists:map(
+        fun(Id) ->
+            <<" zpo", (integer_to_binary(Id))/binary>>
+        end,
+        ObjectIds).
+
```

Comment:
What is an "rtsv"? And "zpo"? (zotonic pivot objects?)

Reply:
These terms come from the zotonic core.

`zpo` is a prefix we use to make the ids into words that are long enough to be uniquely indexed.

`rtsv` is the "related" full text index, where PostgreSQL used `tsv` for the full text indices.

I used the same terms in this es2 module to make it easier to compare the code between Zotonic search and elastic search.

Reply:
Aha thanks for the explanation - good to make it similar 👍